modules/UI/UI.js

```diff
@@ -438,18 +441,31 @@ UI.getSharedVideoManager = function () {
  */
 UI.start = function () {
     document.title = interfaceConfig.APP_NAME;
+
+    return UI.layoutManager.route(window.location.pathname);
+};
+
+
+UI.goToHome = function() {
+    console.log("GOTO: home");
     var setupWelcomePage = null;
-    if(config.enableWelcomePage && window.location.pathname == "/" &&
```

Comment:
It was an my idea. To move "window.location.pathname" in router. And create separate functions for a different layouts rendering.
Now I just divide UI.start function to showWelcomePage and showVideoPage by window.location.pathname

pathname == '/'        --> goToHomePage
pathname == '/[a-Z0-9]'         --> goToVideoPage
pathname == '/test-test or another incorrect path'     --> goToErrorPage